xtext-website/_posts/releasenotes/2020-03-03-version-2-21-0.md

```diff
@@ -8,6 +8,10 @@ published: false
 
 Xtext 2.21.0 is mainly a maintenance release.
 
+## License Change
+
+As requested by the Eclipse Foundation the license for Xtext & Xtend has been changed to [EPL 2.0](https://www.eclipse.org/legal/epl-2.0/).
```

Comment:
I'd prefer something like: `In line with the Eclipse by-laws, Xtext was re-licensed under the EPL 2.0.`
or `According to the Eclipse by-laws ...`

Reply:
weiß nicht ob das da drinne steht

Reply:
wayne points to the FAQ

Reply:
so this would be "In compliance with general requirements by the Eclipse Foundation ....

Reply:
:+1: